source/platforms/nvidia-ai-workbench.md

```diff
@@ -0,0 +1,63 @@
+# NVIDIA AI Workbench
+
+[NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a developer toolkit for data science, machine learning, and AI project development that lets you develop on your laptop/workstation and then move workloads to scalable GPU resources in a data center or the cloud. AI Workbench is free, you can install it in minutes on your local or remote computers, and it provides both a desktop application and a command-line interface (CLI).
```

Comment:
"AI project development that lets you develop".  This reads a bit redundant, and the first sentence is a bit long, how about


```suggestion
[NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a developer toolkit for data science, machine learning, and AI projects. It lets you develop on your laptop/workstation and then easily transition workloads to scalable GPU resources in a data center or the cloud. AI Workbench is free, you can install it in minutes on both local or remote computers, and offers a desktop application  as well as a command-line interface (CLI).
```